tests/testTriggers.go

```diff
@@ -6987,6 +7000,140 @@ func TriggerIopsBwAsyncDR(contexts *[]*scheduler.Context, recordChan *chan *Even
+		allMigrationsSched    = make(map[string]string)
+		includeResourcesFlag  = true
+		includeVolumesFlag    = true
+		startApplicationsFlag = false
+		scpolName             = "async-policy"
+		interval              = MigrationInterval
+		migrationCount        = MigrationsCount
+		suspendSched          = false
+		autoSuspend           = false
+		schd_pol              *storkapi.SchedulePolicy
+		err                   error
+		makeSuspend           = true
+	)
+	chaosLevel := ChaosMap[AsyncDRMigrationSchedule]
+
+	Step(fmt.Sprintf("Deploy applications for migration, with frequency: %v", chaosLevel), func() {
+		err = asyncdr.WriteKubeconfigToFiles()
+		if err != nil {
+			UpdateOutcome(event, fmt.Errorf("failed to write kubeconfig: %v", err))
+			return
+		}
+		err = SetSourceKubeConfig()
+		if err != nil {
+			UpdateOutcome(event, fmt.Errorf("failed to Set source kubeconfig: %v", err))
+			return
+		}
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			taskName := fmt.Sprintf("%s-%d-%s", taskNamePrefix, i, time.Now().Format("15h03m05s"))
+			log.Infof("Task name %s\n", taskName)
+			appContexts := ScheduleApplications(taskName)
+			*contexts = append(*contexts, appContexts...)
+			ValidateApplications(*contexts)
+			for _, ctx := range appContexts {
+				// Override default App readiness time out of 5 mins with 10 mins
+				ctx.ReadinessTimeout = appReadinessTimeout
+				namespace := GetAppNamespace(ctx, taskName)
+				migrationNamespaces = append(migrationNamespaces, namespace)
+			}
+			Step("Create cluster pair between source and destination clusters", func() {
+				// Set cluster context to cluster where torpedo is running
+				ScheduleValidateClusterPair(appContexts[0], false, true, defaultClusterPairDir, false)
+			})
+		}
+		log.InfoD("Migration Namespaces: %v", migrationNamespaces)
+	})
+
+	Step("Create Schedule Policy", func() {
+		schd_pol, err = asyncdr.CreateSchedulePolicy(scpolName, interval)
+		if err != nil {
+			UpdateOutcome(event, fmt.Errorf("schedule policy creation error: %v", err))
+			return
+		} else {
+			log.InfoD("Schedule Policy created with %v mins of interval", interval)
+		}
+	})
+
+	Step("Create Migration Schedule", func() {
+		for i, currMigNamespace := range migrationNamespaces {
+			migrationScheduleName := migrationKey + "schedule-" + fmt.Sprintf("%d", i)
+			currMigSched, createMigSchedErr := asyncdr.CreateMigrationSchedule(
+				migrationScheduleName, currMigNamespace, asyncdr.DefaultClusterPairName, currMigNamespace, &includeVolumesFlag,
+				&includeResourcesFlag, &startApplicationsFlag, schd_pol.Name, &suspendSched, autoSuspend)
+			if createMigSchedErr != nil {
+				UpdateOutcome(event, fmt.Errorf("failed to create migrationschedule wit error %v", err))
+				return
+			}
+			allMigrationsSched[currMigNamespace] = currMigSched.Name
+			time.Sleep(30 * time.Second)
+			mig_sched_resp, err := storkops.Instance().GetMigrationSchedule(currMigSched.Name, currMigNamespace)
```

Comment:
Use camel case for all variables: `migSchedResp`

Reply:
Done!